R/get_ch_name_lookup_path.R

```diff
@@ -0,0 +1,25 @@
+#' Get the full path to the Year Specific Care Home Lookup
+#'
+#' @param year Year of extract
+#' @param ext Extension for the extract (zsav or rds)
+#'
+#' @return Path to clean source extract containing data for each dataset
+#' @export
+get_ch_name_lookup_path <- function(year, ...) {
+  year_dir <- fs::path(
+    "/conf",
+    "sourcedev",
+    "Source_Linkage_File_Updates",
+    year,
+    "Extracts"
```

Comment:
Do we have a function for this dir? If not could we make one, please? Can be done on a separate branch / PR etc.